tt_metal/tools/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/profiler)
+
+set(TOOLS_SRC
+    ${CMAKE_CURRENT_SOURCE_DIR}/memset.cpp)
+
+add_library(tools OBJECT ${TOOLS_SRC})
+target_link_libraries(tools PUBLIC profiler llrt yaml-cpp stdc++fs m)
```

Comment:
More curious, builds failed without yaml-cpp? Also what is `m` ?

Reply:
Originally linked bc it was included in the original .mk, i've removed `yaml-cpp stdc++fs m` and it still builds fine 👍 

`m` is the cpp math library I think